node/tests/SharedTests.ts

```diff
@@ -7706,7 +7706,18 @@ export function runBaseTests(config: {
                 expect(await client.lpos(key, "e")).toBeNull();
 
                 // reverse traversal
-                expect(await client.lpos(key, "b", { rank: -2 })).toEqual(2);
+                expect(
```

Comment:
I think you can collapse it back into a single line, since you have there no changes

Reply:
Updated.